roles/tilemaker/templates/process-openmaptiles.lua

```diff
@@ -246,6 +246,28 @@ function way_function(way)
 		way:Attribute("class", h)
 		SetBrunnelAttributes(way)
 
+        -- Construction
+        local construction = way:Find("construction")
+        if highway == "construction" then
```

Comment:
Could you make this a little more reusable?

Ie. define a variable `constructionValues` and then check for that and add the suffix?

Something like `way:Attribute("class", "motorway_" + tagValue)`? (Don't know the lua syntax for string concatenation)

Reply:
Basically, don't write: `elseif construction == "bla" then` over and over.